background.go

```diff
@@ -9,6 +9,17 @@ import (
 	wmtheme "fyne.io/desktop/theme"
 )
 
+func updateBackgroundPath(background *canvas.Image) {
```

Comment:
This would be better encapsulated (and either to test) if it took the path as a parameter.
Also it would make the updateBackgrounds() call more efficient as it could call wallpaperPath() just once.

Also why the inconsistency between background and wallpaper?